plugins/rsocket-transport/src/main/java/com/github/bsideup/liiklus/transport/rsocket/RSocketLiiklusService.java

```diff
@@ -0,0 +1,58 @@
+package com.github.bsideup.liiklus.transport.rsocket;
+
+import com.github.bsideup.liiklus.protocol.AckRequest;
+import com.github.bsideup.liiklus.protocol.GetEndOffsetsReply;
+import com.github.bsideup.liiklus.protocol.GetEndOffsetsRequest;
+import com.github.bsideup.liiklus.protocol.GetOffsetsReply;
+import com.github.bsideup.liiklus.protocol.GetOffsetsRequest;
+import com.github.bsideup.liiklus.protocol.PublishReply;
+import com.github.bsideup.liiklus.protocol.PublishRequest;
+import com.github.bsideup.liiklus.protocol.ReceiveReply;
+import com.github.bsideup.liiklus.protocol.ReceiveRequest;
+import com.github.bsideup.liiklus.protocol.SubscribeReply;
+import com.github.bsideup.liiklus.protocol.SubscribeRequest;
+import com.github.bsideup.liiklus.service.LiiklusService;
+import com.google.protobuf.Empty;
+import io.netty.buffer.ByteBuf;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@RequiredArgsConstructor
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+public class RSocketLiiklusService implements com.github.bsideup.liiklus.protocol.LiiklusService {
+
+    LiiklusService liiklusServiceImpl;
```

Comment:
```suggestion
    LiiklusService liiklusService;
```
?

Since you have only one always

Reply:
nice leftover finding 👍 Fixed